ha/fault_tolerance/fault_tolerance_driver.py

```diff
@@ -60,24 +58,21 @@ def _get_consumer(self) -> MessageBusConsumer:
 
     def process_message(self, message: str):
         """Callback method for MessageConsumer"""
-        Log.debug(f'Received the message from message bus: {message}')
-        result = False
-        # EventAnalyzer class instantiation is expected.
-        # This is not required. This will be called from EventAnalyzer class.
-        # But as its not available yet and to check newly added filter functionality,
-        # temporarily added this
-        result = self._cluster_resource_filter.filter_event(message.decode('utf-8'))
-        Log.info(f'Alert required: {result}')
+        Log.info(f'Received the message from message bus: {message}')
+        try:
+            EventAnalyzer(message.decode('utf-8'))
+        except Exception as err:
+            Log.error(f'Failed to analyze the event: {err}')
+            return CONSUMER_STATUS.FAILED
         return CONSUMER_STATUS.SUCCESS
 
     def poll(self):
         """Contineously polls for message bus for k8s_event message type"""
         try:
             self._consumer.start()
             while True:
-                # Get alert from message. Analyze changes
-                # with the help of event analyzer filter and publish to message bus
-                # if required
+                # Get alert condition from ALertGenerator. Analyze changes
+                # with the help of event analyzer and notify if required
                 Log.info('Ready to analyze faults in the system')
```

Comment:
this log is not needed inside the while loop

Reply:
Done

Reply:
Thanks. You can add a log before the while loop starts , to indicate the poll is started

Reply:
Closing this comment, we can add the log in a separate PR